docs/MANUAL.md

```diff
@@ -477,6 +477,7 @@ sends a program change command on the current channel. 0000 is program change 1
 **PitchSLide performs a linear pitch slide from previous note value to pitch bb at speed aa**
 - PSL is also time for the first two byte nibble
 - PITCH is linear pitch change
+For MIDI instruments, this plays as portomento the next note (ie. starts the next note *before* stopping this one)
```

Comment:
Why are we reusing a command rather than creating a new one? Wouldn't this command be implemented as a pitch bend in MIDI?

Reply:
Good question @democloid ! I guess mainly to cut down on adding new commands as we only 256 slots and thats shared with all the params we can ever have for any new future instrument types too. But I can make this a new fx command if you would prefer?

But in regards to pitch bend, no my understanding is that this is not how "303 slides" are implemented via MIDI. On the TD-3 and I believe other TB-303 clones (the orginal TB-303 didnt actually have midi afaik) the "slide" that its internal sequencer can do is done in MIDI via sending a note on, then the next note on and then the note off for the first note, ie. "overlapping" the notes, like you would if you were playing via a keyboard and kept holding the first key while you started pressing the second key and then let go of the first. This then causes they bass synth to play the second note without triggering the filter envelope (no volume envelope on these) so as a monosynth, it slides the first note into the second. From what I understand, its basically the essence of the "acid bass" sound.
Experimentally it works on my Behringer TD-3 except for an issue with needing to have at least 1 non-slide note in a sequence, not sure why  that is, I'm consulting with a 303 expert user and will ask on discord as well as I think there are a few expert 303 users there too.